pom.xml

```diff
@@ -136,7 +144,7 @@
             <dependency>
                 <groupId>com.okta.oidc.tck</groupId>
                 <artifactId>okta-oidc-tck</artifactId>
-                <version>0.5.7</version>
+                <version>0.5.8-SNAPSHOT</version>
```

Comment:
This will fail the build, but it should pass locally if you build https://github.com/okta/okta-oidc-tck/pull/110 first